packages/web3-core/types/index.d.ts

```diff
@@ -67,31 +72,37 @@ export interface Web3ModuleOptions {
 }
 
 export interface Providers {
-    HttpProvider: HttpProvider;
-    WebsocketProvider: WebsocketProvider;
-    IpcProvider: IpcProvider;
+    HttpProvider: new (host: string, options?: HttpProviderOptions) => HttpProvider;
```

Comment:
This does not have to get changed because I removed the non-static property. As said before I can do these changes also by myself.

Reply:
This NEEDS to be like this or the typings will not work! web3.providers is a static thing and is stated in the docs

‘const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));’ 

If you don’t under the typings on all of them to this then nobody will ever be able to use the providers static method.